build-scripts/webpack.js

```diff
@@ -2,6 +2,7 @@ const webpack = require("webpack");
 const path = require("path");
 const TerserPlugin = require("terser-webpack-plugin");
 const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
+const CompressionWebpackPlugin = require("compression-webpack-plugin");
```

Comment:
We try to keep as little responsibility in webpack as possible, as we want to limit our exposure to webpack to just being a bundler. Which is why a lot of other tasks are done in gulp. If I recall correctly, it's also taking a long time, although that might have been due to minifying.

Reply:
For day-to-day development, it's not necessary to compress. Hence, I would hesitate to turn this on for everyone.